docs/algorithms.md

```diff
@@ -569,12 +560,29 @@ After some time when the results are ready, you can run the following command to
 ./build/cocos-cli results ./private.pem
 ```
 
+The logs will be similar to this:
+
+```bash
+2024/08/19 14:14:31 Retrieving computation result file
+2024/08/19 14:14:31 Computation result retrieved and saved successfully!
```

Comment:
Add some simple debugging guides. i.e., what error indicates the wrong gRPC port was specified and how to solve it.

Reply:
I don't think this should be covered here since we don't cover edge cases. Documentation is currently based on happy flow

Reply:
agreed debugging guide is in the developer guide.